tests/queries/0_stateless/02724_show_indexes.sql

```diff
@@ -78,3 +78,22 @@ SHOW INDEX FROM database_123456789abcde.tbl;
 DROP DATABASE database_123456789abcde;
 
 DROP TABLE tbl;
+
+DROP TABLE IF EXISTS `tab.with.dots`;
```

Comment:
The right place for this test would have been l. 45 (in section "Check with weird table names"). I guess the (fixed) bug was also not dependent on the existence of indexes, a single column with a single primary key as in l. 37 would have been just fine.

Reply:
I will create a PR to move it